massgov/mayflower-patternlab
# DP-3252 [a11y] Remove IDs from svg icons

## Description

Remove `id` from `<svg>`, `<defs>` and `<path>` from the icons.
Add `aria-hidden="true"` to ones missing `aria-hidden`.  So, they are removed from accessibility tree.

## Related Issue / Ticket

- [DP-3252 A11y Resolve duplicate IDs related to SVG icons](https://jira.state.ma.us/browse/DP-3252)

## Steps to Test


1. In generated pages which contain any svg icons, check their source code if `<svg>`, `<defs>` and `<path>` have `id` attribute.  None of them are supposed to have `id`.
2. Check for any display or function issues.

Tag, `5.10.1-alpha-32522`, is available for testing in Drupal.

## Screenshots
n/a


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
